src/jit/compiler.h

```diff
@@ -1015,11 +1082,20 @@ class CompTimeSummaryInfo
 //
 class JitTimer
 {
-    unsigned __int64 m_start;         // Start of the compilation.
-    unsigned __int64 m_curPhaseStart; // Start of the current phase.
+    unsigned __int64 m_start;           // Start of the compilation.
+    unsigned __int64 m_curPhaseStart;   // Start of the current phase.
+#if MEASURE_CLRAPI_CALLS
+    unsigned __int64 m_CLRcallStart;    // Start of the current CLR API call (if any).
+    unsigned __int64 m_CLRcallInvokes;  // CLR API invokes under current outer so far
+    unsigned __int64 m_CLRcallCycles;   // CLR API  cycles under current outer so far.
+    int              m_CLRcallAPInum;   // The enum/index of the current CLR API call (or -1).
+#endif
 #ifdef DEBUG
     Phases m_lastPhase; // The last phase that was completed (or (Phases)-1 to start).
 #endif
+#if MEASURE_CLRAPI_CALLS
+    static double    s_cyclesPerSec;    // Cached for speedier measurements
+#endif
```

Comment:
There are more things in this one in my local copy (which has many more changes), but when it's the sole survivor it's probably better to combine - will do.